### PR TITLE
Create configurations and src folders only for plugins that have native files

### PR DESCRIPTION
**Pull request by Fatme**

### CODE REVIEW

File: lib/services/android-project-service.ts [hunk 1]
```diff
@@ -296,18 +296,20 @@ export class AndroidProjectService extends projectServiceBaseLib.PlatformProject
...
+				this.$fs.ensureDirectoryExists(pluginConfigurationDirectoryPath).wait();
+
+				// Copy all resources from plugin
+				let resourcesDestinationDirectoryPath = path.join(this.platformData.projectRoot, "src", pluginName);
+				this.$fs.ensureDirectoryExists(resourcesDestinationDirectoryPath).wait();
+				shell.cp("-Rf", path.join(pluginPlatformsFolderPath, "*"), resourcesDestinationDirectoryPath);
+			}
 
 			// Copy include.gradle file
 			let includeGradleFilePath = path.join(pluginPlatformsFolderPath, "include.gradle");
```
Comment (rosen-vladimirov) [inline lib/services/android-project-service.ts]: should we copy this file in case the plugin does not have native code?

**Comment by rosen-vladimirov** [comment]
:+1:

File: lib/services/android-project-service.ts (hunk 1 above)
Comment (Fatme) [inline lib/services/android-project-service.ts]: No, we copy it only when the plugin has platforms/android folder

**Comment by Plamen5kov** [comment]
:+1:

**Comment by vchimev** [comment]
:+1:

**Pull request merged**